src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -157,7 +159,7 @@ private void setWikiTitle(Article article) {
 	 */
 	private void setIsList(Article article) {
 		for (String list : locale.getListIdentifiers()) {
-			if (StringUtils.startsWithIgnoreCase(article.getTitle(), list)) {
+			if (StringUtils.startsWithIgnoreCase(article.getTitle(), list + " ")) {
```

Comment:
I'm trying to minimise the effect of this PR, list change here is unnecessary but I just stumbled upon a bug so I fixed it. This needs lots more work, but it should be a separate project if we want to cover multi-lingual stuff. 

English is main priority for now.